readme.txt

```diff
@@ -291,6 +291,8 @@ At no point during the 3.0 lifecycle will the major version change. But you can
 
 * Fix - replace bad return type to avoid notices in the error log [62376]
 * Fix - add missing styles for screen reader text [62500]
+* Fix - allow '0' to be set as the value of additional fields [45674]
+* Tweak - adjust the month view caching rules [46581]
```

Comment:
I know it is nitpicking.. but should the allow 0 changelog entry be added to Pro and not Core?